service/controller/v25/resource/tccpoutputs/create.go

```diff
@@ -5,11 +5,10 @@ import (
 	"fmt"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/cloudformation"
```

Comment:
This is what I was aiming for. Getting rid of two `cloudformation` packages that are imported in the same file.